build/debian/rules

```diff
@@ -33,6 +33,11 @@ MIXXX_SCONS_FLAGS += install_root=$(CURDIR)/debian/tmp/usr
 %:
 	dh $@ --parallel
 
+# dh_auto_configure will attempt to run cmake instead of scons
+# nothing needs to be done for the configure step; scons is run in the build step below
+override_dh_auto_configure:
+	:
```

Comment:
Or should we try to explicitly specify the build system with `dh_auto_configure --buildsystem scons` (just guessed, not tested) instead of skipping this stage?

Reply:
`dh -l` does not list scons as an automatically supported build system (at least when running it on Fedora 30). Otherwise, I think some of the other overrides in this file would be unnecessary.